src/main/scala/dameo/variant/Variant.scala

```diff
@@ -101,6 +119,16 @@ abstract class Variant private[variant] (
   def defaultRole: Role = Role.defaultRole
 
   def gameFamily: GameFamily
+
+  def updatePositionHashes(
+      board: Board,
+      move: Move,
+      hash: PositionHash
+  ): PositionHash = {
+    val newHash = Hash(Situation(board, !move.piece.player))
+    newHash ++ hash
+  }
+
```

Comment:
I guess we only want to updatePositionHashes on end of turn? Not midturn? But maybe it doesn't matter?